### PR TITLE
Add allure, pytest comment marker for separated tests for eco-news-comment-controller

### PR DESCRIPTION
**Pull request by drozdovskyy**

### CODE REVIEW

**Review by FOleks — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -175,6 +175,9 @@ cython_debug/
 # Discord Rich Presence
 discord.xml
 
+#Allure
+allure-results/
```
Comment (smelovd) [inline .gitignore]: remove, already exist in file

File: tests/api_tests/conftest.py [hunk 2]
```diff
@@ -48,6 +49,7 @@ def test_one(custom_logger):
     TcLogger.generate_logs()
 
 
+@allure.step("Get Authorisation Token")
```
Comment (smelovd) [inline tests/api_tests/conftest.py]: Authorisation -> Authorization

**Review by smelovd — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (drozdovskyy) [reply .gitignore]: Done

File: tests/api_tests/conftest.py (hunk 2 above)
Comment (drozdovskyy) [reply tests/api_tests/conftest.py]: Done

**Review by ol271176 — APPROVED** [review]

**Review by smelovd — APPROVED** [review]

**Pull request merged**